include/aspect/utilities.h

```diff
@@ -74,6 +74,23 @@ namespace aspect
     bool fexists(const std::string &filename);
 
     /**
+     * Loads the content of the ascii file @param filename on process 0 and
+     * distribute the content by MPI_Bcast to all processes. After the call
```

Comment:
distribute -> distributes.

I think I would prefer the name "read_and_distribute_..." -- "load" is a somewhat old-fashioned term.